xaynetwork/xayn_discovery_app
# TB-4173 Replace sourcesChanged event with multiple sourceChanged events

### What 🕵️ 🔍

- Replace `sourcesChanged` event with multiple `sourceChanged` events.
- The issue of using a single `sourcesChanged` event is that we can't have aggregated charts with the properties of `sourcesChanged` and `sourceChanged` event using the default "event count".

----------

### How to test (please adjust template) 🥼 🔬
- open source management screen
- add and remove, trusted and excluded sources
- check mixpanel

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-4173)

----------
